app/models/country.rb

```diff
@@ -1,3 +1,7 @@
 class Country < ApplicationRecord
   has_many :organisations
+
+  scope :with_organisations, -> {
+    joins(:organisations).where('organisations.country_id IS NOT NULL').uniq
```

Comment:
apparently `DEPRECATION WARNING: uniq is deprecated and will be removed from Rails 5.1 (use distinct instead)`